modules/simple_forms_api/app/models/simple_forms_api/vba_20_10207.rb

```diff
@@ -97,7 +98,11 @@ def submission_date_stamps
       []
     end
 
-    def track_user_identity(confirmation_number); end
+    def track_user_identity(confirmation_number)
+      identity = "#{data['preparer_type']} #{data['third_party_type']}"
```

Comment:
is there any possibility of these being undefined?

Reply:
Yes third party type can be `nil` but that would stringify to `""`. As long as we're not calling a method on `nil` itself, it should be ok.